tags:
  - tag_id: 58
    map_segment: "segment_A"
    path:
      - [[0, -1], [1, -1]]
      - [[1, -1], [1, 0]]
  - tag_id: 20
    map_segment: "segment_B"
    path:
      - [[1, 1], [2, 1.5]]
      - [[2, 1.5], [3, 3]]